PixiEditor/Helpers/Converters/FileExtensionToColorConverter.cs

```diff
@@ -7,30 +11,34 @@ namespace PixiEditor.Helpers.Converters
     public class FileExtensionToColorConverter :
         SingleInstanceConverter<FileExtensionToColorConverter>
     {
-        private static readonly SolidColorBrush PixiBrush = ColorBrush(226, 1, 45);
+        private static readonly Dictionary<string, SolidColorBrush> extensions2Brushes;
```

Comment:
Can you put the extension color logic in `SupportedFilesHelper` too please

Reply:
I would leave it for someone doing #331. There should be a model added that would bound file extension to a file color.

Reply:
https://github.com/PixiEditor/PixiEditor/pull/330#issuecomment-1029228687

This would be the best solution for this

Reply:
Also yes, I agree with @CPKreuz. Binding colors in settings is really a minor thing and we will not do it in near future (unless you want to do it).